invenio_oauthclient/handlers/rest.py

```diff
@@ -50,7 +52,7 @@ def default_response_handler(remote, url, payload=None):
 
 def response_handler(remote, url, payload=None):
     """Handle oauthclient rest response."""
-    if not remote:
+    if not remote or not hasattr(remote, 'name'):
```

Comment:
we don't need this `` or not hasattr(remote, 'name')``

Reply:
Then exception will still happen then, as remote is string in our situation and string will be true there

Reply:
There is no use of ``remote.name``  in ``default_response_handler``, the exception is not happenening anymore becuase the code has been changed since we create the issue. The only part that we have to change is to configure our own ``default_response_handler``.